src/dmd/mtype.d

```diff
@@ -1126,6 +1126,14 @@ extern (C++) abstract class Type : RootObject
         return mod == 0;
     }
 
+    /**
+     * Indicates wether the type is usable with `extern(C++)`.
```

Comment:
Whether

Reply:
You'll also be needing a `Returns` documentation line,to follow the ddoc convention.